tests/tests_unit/test_data_classes/test_capabilities.py

```diff
@@ -347,6 +350,40 @@ def test_partly_missing_capabilities(
         )
         assert missing_acls == [has_not]
 
+    def test_raw_acl_database_scope_only(self, cognite_client):
+        # Would fail with: 'ValueError: No capabilities given' prior to 7.2.1 due to a bug in 'as_tuples'.
+        has_all_scope = RawAcl([RawAcl.Action.Read], AllScope)
+        has_db_scope = RawAcl([RawAcl.Action.Read], TableScope(dbs_to_tables={"db1": []}))
+        assert not cognite_client.iam.compare_capabilities(has_all_scope, has_db_scope)
+
+    @pytest.mark.parametrize(
+        "extra_existing",
+        [
+            [],
+            [RawAcl(actions=[RawAcl.Action.Read], scope=AllScope)],
+            [RawAcl(actions=[RawAcl.Action.Read], scope=TableScope({"db1": []}))],
+            [RawAcl(actions=[RawAcl.Action.Read], scope=TableScope({"db1": {"tables": []}}))],
```

Comment:
These are confusing, but if I understand it correctly, it means all in `db1`? Both of them are equal?

Reply:
This means access is granted to all current and all future tables in the database `"db1"`:
`TableScope({"db1": []})`

I don't like the syntax one bit.

Reply:
However, that's how the API does it, and hence we shouldn't add our own `DataBaseScope` imo